Source/NameServer/AddressBook/AddressBook.hpp

```diff
@@ -0,0 +1,165 @@
+/*==============================================================================
+ *  AddressBook: A back-end implementation for the POETS sbase & Nameserver
+ *
+ *  Tasks can be *mapped* and *linked*:
+ *      Mapped: Device Records connected to Device (and Message) and Attribute
+ *              types. Device Records with External Connections also listed.
+ *      Linked: Device Records connected to Addresses in Address Map and listed
+ *              in the relevant Supervisor vector.
+ *
+ *      Linkage is invalidated when/if a Device Record is updated after load.
+ *      Mapping is invalidated if more devices than expected are loaded.
+ *
+ *  Struct sizing with no Pragma Packing:
```

Comment:
This should really be in the docs, and tied to a specific software version - one could easily make a small change and forget to update these figures (particularly if it's me, and particularly if I don't have a Windows machine or Mac).

Reply:
Will be added